### PR TITLE
Doc tweak transform tutorial

### PR DESCRIPTION
**Pull request by tacaswell**
## PR Summary

I tried to clarify some of the transform tutorial prose based on https://github.com/matplotlib/matplotlib/issues/22688#issuecomment-1077791458

## PR Checklist

**Documentation**

- [x] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

File: tutorials/advanced/transforms_tutorial.py [hunk 1]
```diff
@@ -3,80 +3,94 @@
...
-something *other* than the `~.transforms.IdentityTransform()`; the default when
-an artist is placed on an Axes using `~.axes.Axes.add_artist` is for the
-transform to be ``ax.transData``.
++----------------+-----------------------------------+-----------------------------+
+|Coordinate      |Description                        |Transformation object        |
+|system          |                                   |from system to display       |
++================+===================================+=============================+
+|"data"          |The coordinate system for the data,|``ax.transData``             |
+|                |controlled by xlim and ylim.       |                             |
++----------------+-----------------------------------+-----------------------------+
```
Comment (jklymak) [inline tutorials/advanced/transforms_tutorial.py:26]:
```suggestion
|"data"          |The axes coordinate system for the data,|``ax.transData``             |
|                |controlled by xlim and ylim.       |                             |
+----------------+-----------------------------------+-----------------------------+
```

File: tutorials/advanced/transforms_tutorial.py [hunk 2]
```diff
@@ -3,80 +3,94 @@
...
+|                |pts for svg/pdf.                   |                             |
++----------------+-----------------------------------+-----------------------------+
+
+
+
+
+
+The `~matplotlib.transforms.Transform` objects are naive to the source and
+destination coordinate systems, however the objects refereed to in the table
+above are constructed to take inputs in their coordinate system, and transform
```
Comment (jklymak) [inline tutorials/advanced/transforms_tutorial.py:72]:
```suggestion
destination coordinate systems, however the objects referred to in the table
above are constructed to take inputs in their coordinate system, and transform
```

File: tutorials/advanced/transforms_tutorial.py [hunk 3]
```diff
@@ -3,80 +3,94 @@
...
+the input to the *display* coordinate system.  That is why the *display*
+coordinate system has ``None`` for the ``Transformation Object`` column -- it
+already is in *display* coordinates.  The naming and destination conventions
+are an aid to keeping track of the available "standard" coordinate systems and
+transforms.
+
+The transformations also know how to invert themselves, to go from *display*
+back to the native coordinate system.  This is particularly useful when
```
Comment (jklymak) [inline tutorials/advanced/transforms_tutorial.py:80]:
```suggestion
The transformations also know how to invert themselves, to go from *display*
back to the transform's coordinate system.  This is particularly useful when
```

not sure this is right, but "native" isn't fully clear here...

**Review by jklymak — COMMENTED** [review]
This all looks helpful to me, plus or minus a couple of typos/suggestions.

File: tutorials/advanced/transforms_tutorial.py [hunk 4]
```diff
@@ -3,80 +3,94 @@
...
+framework to easily move between coordinate systems, the userland *data*
+coordinate system, the *axes* coordinate system, the *figure* coordinate
+system, and the *display* coordinate system.  In 95% of your plotting, you
+won't need to think about this, as it happens under the hood, but as you push
+the limits of custom figure generation, it helps to have an understanding of
+these objects so you can reuse the existing transformations Matplotlib makes
+available to you, or create your own (see :mod:`matplotlib.transforms`).  The
+table below summarizes the some useful coordinate systems, the description of
+that system, and the transformation object for going from that coordinate
+system to the *display* coordinates.  In the ``Transformation Object`` column,
```
Comment (QuLogic) [inline tutorials/advanced/transforms_tutorial.py:16]:
```suggestion
table below summarizes some useful coordinate systems, a description of
each system, and the transformation object for going from each coordinate
system to the *display* coordinates.  In the ``Transformation Object`` column,
```

File: tutorials/advanced/transforms_tutorial.py (hunk 1 above)
Comment (QuLogic) [reply tutorials/advanced/transforms_tutorial.py:26]: I would say 'The coordinate system for the data in an Axes'

File: tutorials/advanced/transforms_tutorial.py (hunk 4 above)
Comment (QuLogic) [reply tutorials/advanced/transforms_tutorial.py:16]: Also, there is no `Transformation Object` column now, per se.

File: tutorials/advanced/transforms_tutorial.py [hunk 5]
```diff
@@ -3,80 +3,94 @@
...
 ``ax`` is a :class:`~matplotlib.axes.Axes` instance, and ``fig`` is a
 :class:`~matplotlib.figure.Figure` instance.
```
Comment (QuLogic) [inline tutorials/advanced/transforms_tutorial.py:18]: We missed defining `subfigure` before.

File: tutorials/advanced/transforms_tutorial.py [hunk 6]
```diff
@@ -3,80 +3,94 @@
...
+|                |of the figure, and (1, 1) is top   |                             |
+|                |right of the figure.               |                             |
++----------------+-----------------------------------+-----------------------------+
+|"figure-inches" |The coordinate system of the       |``fig.dpi_scale_trans``      |
+|                |`.Figure` in inches; (0, 0) is     |                             |
+|                |bottom left of the figure, and     |                             |
+|                |(width, height) is the top right   |                             |
+|                |of the figure in inches.           |                             |
++----------------+-----------------------------------+-----------------------------+
+|"xaxis",        |Blended coordinate systems; use    |``ax.get_xaxis_transform()``,|
```
Comment (QuLogic) [inline tutorials/advanced/transforms_tutorial.py:50]:
```suggestion
|"xaxis",        |Blended coordinate systems, using  |``ax.get_xaxis_transform()``,|
```

File: tutorials/advanced/transforms_tutorial.py [hunk 7]
```diff
@@ -3,80 +3,94 @@
...
+|"yaxis"         |data coordinates on one of the axis|``ax.get_yaxis_transform()`` |
```
Comment (QuLogic) [inline tutorials/advanced/transforms_tutorial.py:51]:
```suggestion
|"yaxis"         |data coordinates on one direction  |``ax.get_yaxis_transform()`` |
```

File: tutorials/advanced/transforms_tutorial.py [hunk 8]
```diff
@@ -3,80 +3,94 @@
...
+|"display"       |The native coordinate system of the|``None``, or                 |
+|                |output ; (0, 0) is the bottom left |``IdentityTransform()``      |
+|                |of the window, and (width, height) |                             |
+|                |is top right of the output in      |                             |
+|                |"display units".                   |                             |
+|                |                                   |                             |
+|                |The exact interpertation of the    |                             |
+|                |units depends on the back end. For |                             |
+|                |example it is pixels for Agg and   |                             |
+|                |pts for svg/pdf.                   |                             |
```
Comment (QuLogic) [inline tutorials/advanced/transforms_tutorial.py:63]:
```suggestion
|                |points for svg/pdf.                |                             |
```

File: tutorials/advanced/transforms_tutorial.py (hunk 3 above)
Comment (QuLogic) [reply tutorials/advanced/transforms_tutorial.py:80]: back to the input coordinate system?

**Comment by jklymak** [comment]
Moved to draft for now

**Comment by tacaswell** [comment]
Directly took about half of the coments and tried to address the rest.

**Review by jklymak — APPROVED** [review]

**Comment by jklymak** [comment]
Lets merge as an improvement.  If there are further suggestions, they can come in a new PR

**Pull request merged**